pysonde/readers/reader_helpers.py

```diff
@@ -194,10 +195,10 @@ def get_sounding_profile(file, keys):
         warnings.warn("Key {} not found.".format(var), VariableNotFoundInSounding)
     pd_snd = pd.DataFrame.from_dict(sounding_dict, orient="index")
     types = {c: float for c in pd_snd.columns}
-    types["SoundingIdPk"] = str
-    types["DataSrvTime"] = str
-    types["PtuStatus"] = int
-    types["WindInterpolated"] = bool
+    #types["SoundingIdPk"] = str
+    #types["DataSrvTime"] = str
+    #types["WindInterpolated"] = bool
+    ##types["PtuStatus"] = int
```

Comment:
This shouldn't be a problem if the values are not given in the config file. For the cor files I deleted the entire section in the config.